docs/integrations/sources/chargebee.md

```diff
@@ -104,7 +104,8 @@ The Chargebee connector should not run into [Chargebee API](https://apidocs.char
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                                |
 |:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 0.9.0  | 2024-10-22 | [47243](https://github.com/airbytehq/airbyte/pull/47243) | Migrate to Manifest-only  |
+| 0.9.1   | 2025-07-02 | [53220](https://github.com/airbytehq/airbyte/pull/53220) | Replace custom `IncrementalSingleSliceCursor` component with low-code semi-incremental `DatetimeBasedCursor`   |
```

Comment:
nit, but let's make this a minor version bump to `0.10.0`. This change will make the modified streams processed concurrently with each other for better sync performance so I think that would be considered a feature vs. a patch.